polling_stations/apps/data_collection/management/commands/import_rochdale.py

```diff
@@ -4,21 +4,15 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E08000005"
-    addresses_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019Rochdale.tsv"
-    stations_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019Rochdale.tsv"
-    elections = ["local.2019-05-02"]
+    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019.tsv"
+    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019.tsv"
+    elections = ["parl.2019-12-12"]
     csv_delimiter = "\t"
+    allow_station_point_from_postcode = False
 
     def station_record_to_dict(self, record):
 
-        # Mobile Unit at Black Dog Pub
-        if record.polling_place_id == "2829":
-            record = record._replace(polling_place_postcode="OL12 7JG")
-            rec = super().station_record_to_dict(record)
-            rec["location"] = Point(-2.186986, 53.631252, srid=4326)
-            return rec
-
-        # Bowls Pavilion
+        # Bowls Pavilion - Postcode gives wrong geocode
         if record.polling_place_id == "2577":
```

Comment:
Is this definitely ID 2577 again?
Usually the IDs change between elections with Xpress

Reply:
Good catch. Thanks.